### PR TITLE
BMCP-2: Some basic folder and files

### PR DESCRIPTION
**Pull request by nikhilsharma0025**
## Description
- Added some components and pages folder and created basic components file like topbar, footer and routes

#### Modification
-added components  and pages folder 
-added  routes,footer , topbar file

### CODE REVIEW

File: src/components/footer/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function footer() {
```
Comment (nik-1236789) [inline src/components/footer/footer.tsx:3]: Consider starting the component name with capital alphabets and try using ES6 syntax (optional) for function creation along with  proper types (required) from "typescript" .

File: src/components/topbar/topbar.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function topbar() {
```
Comment (nik-1236789) [inline src/components/topbar/topbar.tsx:3]: Same as in Footer component

File: src/routes.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import Home from "./pages/Home";
+
+const Routes = () => {
+    return (
+        <Switch>
```
Comment (nik-1236789) [inline src/routes.tsx:7]: You need to wrap the routes inside a Router. Consider using BrowserRouter or HashRouter(preferred). Otherwise this won't work.

**Review by nik-1236789 — CHANGES_REQUESTED** [review]
Have a look on the points mentioned in the comments.

File: src/components/footer/footer.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    return (
+        <div>
```
Comment (2SkeePeR2) [inline src/components/footer/footer.tsx:5]: Consider using ```React.Fragment``` for such components.

File: src/components/topbar/topbar.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    return (
+        <div>
```
Comment (2SkeePeR2) [inline src/components/topbar/topbar.tsx:5]: ```React.Fragment```

File: src/pages/home.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Home = () => {
+    return (
+        <div>
```
Comment (2SkeePeR2) [inline src/pages/home.tsx:5]: ```React.Fragment```

**Review by 2SkeePeR2 — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 7]
```diff
@@ -1,16 +1,15 @@
...
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import { HashRouter } from 'react-router-dom';
+import { App } from './App';
 import reportWebVitals from './reportWebVitals';
 
+
 ReactDOM.render(
-  <React.StrictMode>
```
Comment (2SkeePeR2) [inline src/index.js:8]: Don't remove this tag

File: src/react-app-env.d.ts [hunk 8]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (2SkeePeR2) [inline src/react-app-env.d.ts:1]: Why comment ?

**Review by 2SkeePeR2 — CHANGES_REQUESTED** [review]

**Review by nik-1236789 — APPROVED** [review]

**Review by nik-1236789 — APPROVED** [review]

**Pull request merged**